src/scripts/main.js

```diff
@@ -168,6 +168,15 @@
         return Object.values(mice);
     }
 
+    function getMapItems(data) {
+        const items = {};
+        $.each(data.treasure_map.goals.item, (key, item) => {
+            items[item.unique_id] = item.name;
+        });
+
+        return Object.values(items);
```

Comment:
The only need for the dict approach was to support removing caught mice by the id--if that's not needed then this is more to the point:
```suggestion
        return Object.values(data.treasure_map.goals.item)
          .map((item) => item.name);
```